corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -339,8 +400,200 @@ def _get_table_size(config):
             for datasource in static_datasources + dynamic_datasources
         ], key=lambda r: r[-1] if r[-1] != 'Table not found' else 0)
 
+        ResourceModel.set_stat(ResourceModel.DATASOURCES_INFO, rows)
+
+    def _case_transactions(self):
+        db_name = get_db_aliases_for_partitioned_query()[0]
+        db_cursor = connections[db_name].cursor()
+
+        with db_cursor as cursor:
+            cursor.execute("""
+                SELECT COUNT(*) as num_forms, d.count as num_updates
+                FROM (
+                    SELECT COUNT(*) as count
+                    FROM form_processor_casetransaction
+                    WHERE case_id IN (
+                        SELECT case_id
+                        FROM form_processor_commcarecasesql
+                        WHERE domain = %s
+                    )
+                    GROUP BY form_id
+                ) AS d
+                GROUP BY d.count;
+            """, [self.domain])
+            result = cursor.fetchall()
+
+            running_form_case_updates = 0
+            total_forms = 0
+            for num_forms, num_updates in result:
+                total_forms += num_forms
+                running_form_case_updates += num_forms * num_updates
+
+            ResourceModel.set_stat(ResourceModel.CASE_TRANSACTIONS_FACTOR, running_form_case_updates / total_forms)
+
+            cursor.execute("""
+                SELECT COUNT(*)
+                FROM form_processor_casetransaction
+                WHERE case_id IN (
+                    SELECT case_id
+                    FROM form_processor_commcarecasesql
+                    WHERE domain = %s
+                );
+            """, [self.domain])
+            (total_transactions,) = cursor.fetchone()
+            ResourceModel.set_stat(ResourceModel.CASE_TRANSACTIONS_TOTAL, total_transactions)
+
+    def _case_indices(self):
+        db_name = get_db_aliases_for_partitioned_query()[0]
+        db_cursor = connections[db_name].cursor()
+
+        with db_cursor as cursor:
+            cursor.execute("""
+                SELECT COUNT(*)
+                FROM form_processor_commcarecaseindexsql
+                WHERE domain = %s;
+            """, [self.domain])
+            (total_case_indices,) = cursor.fetchone()
+
+            total_cases = ResourceModel.get_stat(ResourceModel.CASES_TOTAL)
+            if total_cases > 0:
+                ResourceModel.set_stat(ResourceModel.CASE_INDICES, total_case_indices/total_cases)
+
```

Comment:
I think it can definitely skew things but it really depends on what you want the data for. I guess it would probably be most useful to have both (per case type and average).